README.md

```diff
@@ -808,6 +808,77 @@ Currently there is no CLI for this, though more information can be found about t
 
 &nbsp;
 
+
+## Transaction priorities
+
+`ChainBridge` supports making deposits with different transaction speeds. To utilize those you can
+pass priority flag [`slow`/`medium`/ `fast`] when executing a [deposit](#deposit).
+Transaction priorities can be implemented via `ITX` or `metadata priority`, which will be clarified below. The following example show how to utilize the `priority` flag.
+
+```bash
+erc20 deposit
+  --url=**'node endpoint'**
+  --private-key=**'bridge admin private key'**
+  --amount=1
+  --decimals=18
+  --bridge=**'bridge contract address'**
+  --recipient=**'recipient address'**
+  --domain=**'domain id'**
+  --resource=**'registered resource'**
+  --priority=slow
+```
+### `ITX`
+[Infura Transactions (ITX)](https://blog.infura.io/how-to-use-itx-step-by-step-guide/) is the simplest way to send transactions on Ethereum because if handles all edge cases for sending transactions and spares users of gas management complexities. Also a notable benefit is that it allows dapp users to send transactions without holding ETH.
```

Comment:
```suggestion
[Infura Transactions (ITX)](https://blog.infura.io/how-to-use-itx-step-by-step-guide/) is the simplest way to send transactions on Ethereum because it handles all edge cases for sending transactions and spares users of gas management complexities. Also, a notable benefit is that it allows dapp users to send transactions without holding ETH.
```